README.md

```diff
@@ -106,7 +106,7 @@ The following table explains the different configuration parameters and how to s
 | `kamon.jmx-collector.mbeans[*].metric-name`        | The name of the Kamon metric instrument that is instantiated                                                                                                                   | *(Any)*                                                                                | `memory-consumption`     |
 | `kamon.jmx-collector.mbeans[*].jmxQuery`           | The query for the JMX object name                                                                                                                                              | *(Any)*                                                                                | `java.lang:type=Memory`  |
 | `kamon.jmx-collector.mbeans[*].attributes[*].name` | The name of one of the attributes under the JMX object                                                                                                                         | *(Any)*                                                                                | `HeapMemoryUsage`        |
-| `kamon.jmx-collector.mbeans[*].attributes[*].type` | The type of the Kamon [metric intrument](http://kamon.io/documentation/1.x/core/advanced/metric-instruments/)                                                                  | `counter`, `histogram`, `incrementing-gauge`, `decrementing-gauge`, `punctual-gauge`   | `histogram`              |
+| `kamon.jmx-collector.mbeans[*].attributes[*].type` | The type of the Kamon [metric intrument](http://kamon.io/documentation/1.x/core/advanced/metric-instruments/)                                                                  | `counter`, `histogram`, `incrementing-range-sampler`, `decrementing-range-sampler`, `punctual-gauge`   | `histogram`              |
```

Comment:
According to the docs Gauages are still there; I don't think we should remove it from this library

https://kamon.io/docs/latest/core/advanced/metric-instruments/#gauges

Reply:
yes, but the increment and decrement operations were removed. I added them back in https://github.com/kamon-io/Kamon/pull/596 so lets wait

Reply:
reverted these changes...